### DISCUSSION TITLE
企业如何实践开源协同 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#31] [Comments]
# 企业如何实践开源协同 | 夜天之书

随着开源概念的红火，越来越多的企业将内部项目公开托管到 GitHub 等平台，也有越来越多依托开源项目建立起来的企业。对于这些企业来说，它们的目标不只是开放项目源代码，更希望能够形成开源共同体，打造围绕项目的软件生态。 然而，其中大部分项目由于成员背景的单一性，最终都终结于仅源码可得的形态。对于这些新兴项目来说，初始成员从属于同一企业是既定事实。在这样的前提下，企业应该如何实践开源协同，形成开源共

https://www-tisonkun-org.translate.goog/2021/12/05/open-source-collaboration-enterprise-perspective/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en

**Comment by klDen** [comment]
Thanks a lot for this write up Tison. It is definitely an interesting read.
